### DISCUSSION TITLE
Rollback screen improvements

### DISCUSSION THREAD

**Discussion by mt-bt** [coollabsio/coolify#3266] [Improvement Requests]
Love the feature, here are some things I wish I could do from the rollback screen:
- Being able to see the commit of each image (similar to what we see in the deployments page), right now the SHA is not that helpful
- Remove older images manually in a simple click

**Comment by ksaitor** [comment]
- [ ] Move Rollback functionality from Configuration to Deployments tab
- [ ] In Configuration -> Rollback, users should be able to config number of containers to keep at any time, for instant rollback

![image](https://github.com/user-attachments/assets/6deedbf8-3ba3-436e-8111-61f4ad50f629)

Rollback button example:
![image](https://github.com/user-attachments/assets/d4440161-6171-45f5-b6e8-f2143ab3804b)

**Comment by ksaitor** [comment]
Looks like no one has looked at this issue. Very UI simple fix, imho. But big UX improvement. 
Also, "Rollback" actions are currently under "Configuration" tab, yet there is no rollback config there at all. Does not make logical sense.

cc @andrasbacsai
